script.js

```diff
@@ -0,0 +1,61 @@
+// Задание 1
+// Создаем переменную типа данных массив.Записываем в массив элементы с помощью цикла.
+let arr1 = [];
+for(i = 1;i <= 100; i++){
+    arr1.push( Math.round( Math.random() * 100 ))
+}
+//Создаем новую переменную в которой будет хранится новый массив
+let arr2; 
+//Создаем функцию которая фильтрует и сортирует значения массива arr1 и записывает значения в пременную arr2.
+func1 = () => {
+    arr2 = arr1.filter(el => el > 50).sort((a,b) => a - b);
+}
+func1(arr1);
+console.log(arr2);
+
+// Задание 2
+
+let arr3 = [];  
+let abc = "abcdefghijklmnopqrstuvwxyz";  
+// Создаем функцию которая создает рандомную по длине строку 
+function word (){  
+    const a = [];  
+    const len = Math.ceil(Math.random() * 10) 
+    for(let i = 0;i < len; i++){  
+        a.push(abc[Math.floor(Math.random() * abc.length)]);  
+    }  
+    return a.join(''); // Преобразовуем все элементы массива в одну строку 
+}  
+  
+for(let i = 1;i <= 100;i++) { // Записываем 100 элементов массива рандомными строками 
+    arr3.push(word());  
+} 
+
+const arr4 = arr3.sort().map(value => ({ value }));// Сортируем и создаем новый массив с объектами
```

Comment:
Sort is a dirty function, so you should cope an initial array or (better way) swap for map and sort functions